### ISSUE TITLE
Conflicting information on certificate renewal limit in app

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-documentation#900]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug affects both Android **and** iOS, for specific issues / questions that apply only to one operating system, please raise them in the respective repositories:
    * [Android repository](https://github.com/corona-warn-app/cwa-app-android)
    * [iOS repository](https://github.com/corona-warn-app/cwa-app-ios)
* [X] Bug is not already reported in another issue - ~I remember that somewhere, I think it was @MikeMcC399, mentioned this already~

## Technical details

- Device name: iPhone XR / Huawei P Smart 2019
- iOS/Android version: iOS 16.0 / Android 10
- App version: 2.23.2 / 2.23.2

## Describe the bug

In the app there is conflicting information on the limit how often a certificate can be renewed.

Here in the info text it says only once:

<img src="https://user-images.githubusercontent.com/67682506/174731350-55cee66c-c912-4460-9802-286741b764f5.jpeg" width="300">

But then, if there is an error due to the rate limit, it says 3 times:

<img src="https://user-images.githubusercontent.com/67682506/174731514-79fedd05-3ef8-45ba-9186-330500dc2757.jpeg" width="300">

## Steps to reproduce the issue

1. Start the renewal process for a certificate which has already been renewed more than 3 times
2. See the information text
3. Click on "Einverstanden" 
4. See the error text which conflicts the information text

## Expected behaviour

First it needs to be clarified if one renewal is the limit or if it's possible to renew one certificate up to 3 times.

## Possible Fix

Change the text.

---
Internal Tracking ID: [EXPOSUREAPP-13423](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13423)

**Comment by MikeMcC399** [comment]
@Ein-Tim 
> Bug is not already reported in another issue - I remember that somewhere, I think it was @MikeMcC399, mentioned this already

I have not reported this inconsistency anywhere else.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

- I opened a new issue https://github.com/corona-warn-app/cwa-website/issues/2970 for the website

**Comment by MikeMcC399** [comment]
@Ein-Tim 

> First it needs to be clarified if one renewal is the limit or if it's possible to renew one certificate up to 3 times.

The [blog](https://www.coronawarn.app/en/blog/2022-06-01-cwa-2-23/) is clear about this point:

"A technically expired certificate can only be renewed three times. If users have stored their certificates in the CWA on several devices, they can only renew it on three of them. However, they have the option to export the newly issued certificate as a PDF file so that they can re-scan the QR code of the certificate if needed. "

**Comment by dsarkar** [comment]
Good morning @MikeMcC399 @Ein-Tim! 

Thanks for reporting, Internal Tracking ID: [EXPOSUREAPP-13423](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13423.

Best wishes, DS

---
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Thanks for clarifying & opening the other issue. 

@dsarkar Thank you!

**Comment by Ein-Tim** [comment]
@dsarkar 

I'd be happy to provide a PR for this under iOS. Let me know if this would be appreciated.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

So it seems this will be fixed in version 2.25 (which needs new label creating).

**Comment by Ein-Tim** [comment]
This issue has been fixed in version 2.25. I can confirm the fix. Closing.
